### ISSUE TITLE
Addons installed via one source switch to another

### ISSUE THREAD

**Issue by NCC-Lykos** [AcidWeb/CurseBreaker#285]
I've installed BenikUI and Shadow and Light via tu:11 and tu:38, however during updates and export they switch to using CurseForge. Almost always the addons are updated on Tukui before they are approved on CF. This doesn't seem to be an issue with any addons except Benik and Shadow and Light for some reason... Is there a way that when an addon is installed via one source, it always updates and exports via that source?

**Comment by AcidWeb** [comment]
CB don't support source switching. Update proces can't change the source - it is technically impossible.

Even if you execute `install import` command it will not overwrite them with CF versions.

**Comment by NCC-Lykos** [comment]
I will reply to this when an update is released for one of these addons. I have installed via the tu:## method always and it always switches to using cf: -- This has been an issue for me for awhile and I will report back when I can explain the steps taken to reproduce.

**Comment by AcidWeb** [comment]
Switches where?

**Comment by NCC-Lykos** [comment]
I just went to reinstall WoW so I did an export in CurseBreaker so I could reinstall my addons, it showed BenikUI and Shadow and Light as cf: entries and not tu: entries. I've had this issue in the past when exporting after the addons had been installed for awhile. I just did a test like you did above and it exported as tu: properly, so maybe something changed along the line but I've always run into this problem before now. 

Once I can reproduce properly I will reopen and update, but until then we can consider this as cannot reproduce.
